client-app/src/pages/account/store/actions.ts

```diff
@@ -1,22 +1,26 @@
-import AccountService from "@common/account.service";
+import Vue from "vue";
 import { ActionTree } from "vuex";
 import { FETCH_PROFILE, SET_PROFILE, UPDATE_USER } from "@account/store/definitions";
 import { State } from "@account/store/state";
+import { ApiAccountClient, UserUpdateInfo } from '@common/api/api.client';
 
 //actions
 export const actions: ActionTree<State, State> = {
   async [FETCH_PROFILE](context) {
     try {
       context.commit(FETCH_PROFILE);
-      const { data } = await AccountService.getUserAccount();
-      context.commit(SET_PROFILE, data);
-      return data;
+      const accountClient = new ApiAccountClient(Vue.axios.defaults.baseURL, Vue.axios);
```

Comment:
Why we can't use baseUrl as well as for storeName and currrentLng?

Reply:
Yes, we can. It will be done.